docs/federate.md

```diff
@@ -119,20 +120,20 @@ NB_API_TAG=latest
 
 # Configuration for query tool
 # Define the URL of the f-API as it will appear to a user
-API_QUERY_URL=http://localhost:8080 # (1)!
+API_QUERY_URL=http://206.12.85.19:8080 # (1)!
```

Comment:
Actually, `localhost` doesn't work here! This was one of the [discoveries](https://github.com/neurobagel/planning/issues/111#issuecomment-1955461496) from the BIC internal node deployment (although I also confirmed it on the `st-viateur` machine). Specifically:

>  ### On `neurobagel-node` and `st-viateur`:
> 
> (Query tool)
> 1. When our `local_federation` Compose recipe is used, the resulting query tool cannot access the f-API when `API_QUERY_URL` is set to `localhost:<fAPI port on host>`, even though this is the default recommended in https://neurobagel.org/federate/#fedenv
>   - also doesn't work: using the container name `federation` with the internal port, or `host.docker.internal` with the published port
>   - Instead, `API_QUERY_URL` must be set to the **public IP** of the machine on which the f-API is running (followed by the `<fAPI port on host>`), or the subdomain for the f-API if NGINX is set up on the machine

I think this might be related to https://github.com/neurobagel/query-tool/pull/343.

Reply:
OK, Thanks for explaining!